Tests/MLXTests/PoolingTests.swift

```diff
@@ -0,0 +1,50 @@
+// Copyright © 2024 Apple Inc.
+
+import Foundation
+import XCTest
+import MLX
+import MLXNN
+
+class MLXNNPoolingTests: XCTestCase {
```

Comment:
Tests look good -- please run `pre-commit run --all-files` (it just runs swift-format with our config) on all the files.